core/model/modx/processors/system/log/getlist.class.php

```diff
@@ -69,6 +71,8 @@ public function getData() {
         /* check filters */
         $wa = array();
         if (!empty($actionType)) { $wa['action:LIKE'] = '%'.$actionType.'%'; }
+        if (!empty($classKey)) { $wa['classKey:LIKE'] = '%'.$classKey.'%'; }
```

Comment:
Does this have to be a LIKE? Are there situations where the class key isn't exactly what was chosen?